cdc/kv/metrics.go

```diff
@@ -143,6 +143,21 @@ var (
 		},
 		// actions: wait, run.
 		[]string{"namespace", "changefeed"})
+
+	workerBusyRatio = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: "ticdc",
+			Subsystem: "kvclient",
+			Name:      "region_worker_busy_ratio",
```

Comment:
Should be renamed to kvclient_receiver_busy_ratio.

Reply:
The monitor also records information about the eventHandler worker.